Quaver.Shared/Screens/Gameplay/UI/PauseScreen.cs

```diff
@@ -233,6 +279,13 @@ public override void Update(GameTime gameTime)
             if (Screen.Failed)
                 Visible = false;
 
+            if (Screen.IsPaused)
```

Comment:
`if (Screen.IsPaused && DialogManager.Dialogs.Count == 0)`

This will handle the cases of when dialog screens are open - such as the in-game chat.